src/main/java/com/minecolonies/coremod/event/ClientEventHandler.java

```diff
@@ -67,11 +66,13 @@ public void renderWorldLastEvent(@NotNull final RenderWorldLastEvent event)
             final BlockPos position = Settings.instance.getPosition();
             if (Settings.instance.getStructureName().contains(AbstractEntityAIStructure.WAYPOINT_STRING))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this conditional structure or edit its code blocks so that they're not all the same. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3923)